k8s/apps/bases/frontend/deployment.yaml

```diff
@@ -29,4 +33,14 @@ spec:
             requests:
               cpu: 200m
               memory: 45Mi
+          securityContext:
+            runAsGroup: 1337
+            runAsUser: 1337
```

Comment:
istio apparently doesn't like sharing UIDs. Works for us right now, but something something iptables something something outgoing traffic